public/stylesheets/site/2.0/11-group-listbox.css

```diff
@@ -22,7 +22,7 @@ http://otwcode.github.com/docs/front_end_coding/patterns/listbox
 
 .listbox > .heading, .listbox .heading a:visited {
   margin: 0;
-  color: #222;
+  color: #2a2a2a;
```

Comment:
We don't actually use `#222` anywhere else, which became evident while collecting elements with like colors for the wizard skin.